src/server/rpc/procedures/eclipse-2017/eclipse-2017.js

```diff
@@ -116,51 +116,19 @@ let stationInfo = function(stationId){
         });
 };
 
-let temperature = function(stationId){
-    return _stationReading(stationId).then(reading => {
-        return reading.temp;
-    });
-};
-
-
 let pastTemperature = function(stationId, time){
     return _stationReading(stationId, time).then(reading => {
         return reading.temp;
     });
 };
 
 
-let condition = function(stationId){
-    return _stationReading(stationId).then(reading => {
-        return rpcUtils.jsonToSnapList(hideDBAttrs(reading));
-    });
-};
-
-
 let pastCondition = function(stationId, time){
```

Comment:
It would be nice to rename the `pastCondition` and `pastTemperature` rpcs to just be `temperature` and `condition` (as long as no other projects are using these rpcs).

Reply:
should that be another PR?

Reply:
pastTemperature is being used. we could use the compatibility option. The caveat is that the new name is what we are removing so I think it might confuse users.

Reply:
This only matters if there are users using this service externally. We should check if any of the projects are using this service outside of our demo projects

Reply:
if we can be sure that no one is using it we can, I'm not sure if we can account for netsblox instances other than the one at editor.netsblox.org. Doing the unnecessary and breaking change in another PR keeps this one from having to deal with that issue.